client/login/magic-login/request-login-email-form.jsx

```diff
@@ -49,10 +48,11 @@ class RequestLoginEmailForm extends Component {
 		usernameOrEmail: this.props.userEmail || '',
 	};
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if ( ! this.props.requestError && nextProps.requestError ) {
-			defer( () => this.usernameOrEmail && this.usernameOrEmail.focus() );
+	usernameOrEmailRef = createRef();
+
+	componentDidUpdate( prevProps ) {
+		if ( ! prevProps.requestError && this.props.requestError ) {
+			this.usernameOrEmailRef.current?.focus();
```

Comment:
This is similar behavior as with the`autoFocus` prop. Generally, we should avoid that for accessibility reasons, but it would be a job for another PR, as this component was already doing it.

Reply:
So we should remove all the auto-focusing and re-focusing code altogether from this UI? If we agree it's a good idea, I'd do in this PR as another commit, it's a welcome simplification.

Reply:
Yeah, I think removing the auto-focusing would actually be an improvement 👍